scripts/character_test.js

```diff
@@ -22,7 +24,9 @@ function testChar(character) {
 
 // return false if text contains any bad characters
 function testText(text) {
-    text = text.normalize("NFC");
+    if("normalize" in String.prototype) {
```

Comment:
I've seen `if(String.prototype.normalize)` elsewhere on the internet for this check -- which seems a bit more straightforward.

For all of these, please add a comment `// IE HACK - IE11 does not support string normalization` above them so we can go in and remove them when we eventually drop IE11 support.